### DISCUSSION TITLE
The Upgrade Guide wasn't updated on Next 10

### DISCUSSION THREAD

**Discussion by lucas-varela** [vercel/next.js#18742] [Help] [answered]
The current [upgrade guide](https://nextjs.org/docs/upgrading) page wasn't updated when Next 10 released. Even if no breaking changes were made, I would assume the guide would be updated to reflect that. My team support several projects on Next 9 which we don't feel safe enough to upgrade without knowing if there were breaking changes, specially because React was updated to v17.

Are there any plans to update the guide?

**Comment by jamesmosier** [comment, marked as answer]
Hi @lucas-varela. Because there were no breaking changes I believe that is why there is no migration guide. 

If you wanted a full rundown of new features/changes, see [the release notes](https://github.com/vercel/next.js/releases/tag/v10.0.0) or [the blog post](https://nextjs.org/blog/next-10).

Also React v17 is not required with Next 10 and there is no requirement to use it.

**Reply by lucas-varela** [reply]
Thanks for the response. I still suggest adding this information to the upgrade guide, just in case, since there's no hint that Next 10 was released in there, which made me wonder if there were breaking changes. I would gladly make a PR if you think this could be useful.
